bindings/matlab/Visualization/getMeshes.m

```diff
@@ -0,0 +1,108 @@
+function [linkMeshInfo,map]=getMeshes(model,meshFilePrefix)
+% We use the iDyntree information to obtain the files containing the meshes
+% and we link them to their link names
+% Gets the mesh information for each link in the model.
+%     - Iputs:
+%         - `model` : iDyntree model loaded form a URDF.
+%         - `meshFilePrefix` : Path in which we can find the meshes. As an example the path to the mesh in a iCub urdf is `'package://iCub/meshes/simmechanics/sim_sea_2-5_root_link_prt-binary.stl'
+% `. `meshFilePrefix` should replace package to allow to find the rest of the path.
+%     - Outputs:
+%         - `map`        : Cell array having both the names of the meshes and the associated link
+%         - `linkMeshInfo` : Struct array that contain the link name and a struct (`meshInfo`) that contains the name of file or if is a simple geometry, the triangulation ( edges and vertices of the mesh ) and the link to geometry transform.
+%
+% Author : Francisco Andrade (franciscojavier.andradechavez@iit.it)
+%
+% Copyright (C) 2019 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+% This software may be modified and distributed under the terms of the
+% GNU Lesser General Public License v2.1 or any later version.
+
+
+
+% get the linkSolidShapes containing the mesh information
+visual=model.visualSolidShapes;
+linkSolidShapesV=visual.linkSolidShapes;
+% get number of links
+numberOfLinks=linkSolidShapesV.size;
+linkMeshInfo=struct('meshInfo',{},'linkName',{});
+% get pointer to beggining of vector
+iterator=linkSolidShapesV.begin;
+% iterate getting the name of the mesh and the name of the link
+count=1;
+link_with_no_visual=[];
+for links=1:numberOfLinks    
+    linkName=model.getLinkName(links-1);
+    linkMeshInfo(links).linkName=linkName;
+    solidarray=iterator.next;
+    solids_number=size(solidarray,2);    
+    meshInfo=struct('meshFile',{},'mesh_triangles',{},'link_H_geom',{});
+    for solids=1:solids_number
+        if solidarray{solids}.isExternalMesh
+            meshName=split(solidarray{solids}.asExternalMesh.filename,':');
+            meshFile=meshName{2};
+            % Import an STL mesh, returning a PATCH-compatible face-vertex structure
+            mesh_triangles = stlread([meshFilePrefix meshFile]);
+            meshInfo(solids).meshFile=meshFile;
```

Comment:
URDFs (and so iDynTree::Model) support meshes also in collada (dae) format, and we use them a lot for example in  the new iRonCub model. Not supporting dae is ok, but it should be properly documented.

Reply:
I'll have to check if Matlab loads far similarly to STL maybe both can be supported. I guess it will be needed to check the termination of the file.

Reply:
I am afraid that it will not be as easy as STL as collada is much more complex format (so I think just saying that we do not support it should  be enough).

Reply:
I found this https://it.mathworks.com/matlabcentral/fileexchange/45687-colladaparser I can give it a try. I will deal with the other changes and mention that for now we do not support saw. But maybe with this we might able to support it later. If it's ok with you I want to release this without DAE support and then if possible add it in another PR.

Reply:
Of course MATLAB supports DAE in one or 2 of it's toolboxes...

Reply:
>  it's ok with you I want to release this without DAE support and then if possible add it in another PR.

Yes, not having DAE support is ok, as long as it is documented.

Reply:
I created this issue https://github.com/robotology/idyntree/issues/670 for that, and will add documentation on missing .dae support for now